### DISCUSSION TITLE
Search Functionality

### DISCUSSION THREAD

**Discussion by tejasri555** [raghusaripalli/GatorGiveaway#131] [Ideas]
Hello everyone! When looking for a specific item, it may be tough for the user to go through each post. Can we have search functionality?

**Comment by raghusaripalli** [comment]
That's a great idea! We could introduce tags for the posts, so it's easy to group and filter posts based on tags.

**Comment by tejasri555** [comment]
"Tags" is a great idea. Loved it.
Also, we can use the Trie data structure which is efficient in flirting with the tags. 
Creating an issue for it.
